mobile/src/beta/res/values-yi/strings.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
@kaikreuzer Can you sync the project? Crowdin should name this file `.../values-ji/...`

Reply:
Crowdin added files with the mapped languages codes, but didn't remove the old files.
I can try to merge this, remove the l10n branch and you sync again. If crowdin doesn't remove those files, I can do it manually.